MVP Module/Tests/Feed UI/FeedUIIntegrationTests.swift

```diff
@@ -46,6 +46,23 @@ final class FeedUIIntegrationTests: XCTestCase {
 		loader.completeFeedLoadingWithError(at: 1)
 		XCTAssertFalse(sut.isShowingLoadingIndicator, "Expected no loading indicator once user initiated loading completes with error")
 	}
+    
+    func test_loadFeedCompletion_displaysErrorMessageOnError() {
+        let (sut, loader) = makeSUT()
+        
+        sut.loadViewIfNeeded()
+        XCTAssertNil(sut.errorMessage, "Expected no error message once view is loaded")
+        
+        loader.completeFeedLoading(at: 0)
+        XCTAssertNil(sut.errorMessage, "Expected no error message once loading completes successfully")
+        
+        sut.simulateUserInitiatedFeedReload()
+        loader.completeFeedLoadingWithError(at: 1)
+        XCTAssertEqual(sut.errorMessage, "Couldn't connect to server", "Expected error message once user initiated loading completes with error")
```

Comment:
This test seems fragile. It could fail if running in a different locale (e.g., Portuguese) because it's using the hard-coded English string. You could use the [localized test helper method](https://github.com/essentialdevelopercom/ios-lead-essentials-ui-design-patterns-challenge/blob/master/MVP%20Module/Tests/Feed%20UI/Helpers/FeedUIIntegrationTests%2BLocalization.swift) to check by the localized `key` instead of the `value`: 

```suggestion
        XCTAssertEqual(sut.errorMessage, localized("FEED_VIEW_CONNECTION_ERROR"), "Expected error message once user initiated loading completes with error")
```